src/components/Boxs/BoxBase/index.tsx

```diff
@@ -116,7 +116,7 @@ const BoxBase: React.FC<BoxBaseProps> = ({
         color = color || "text.primary";
     }
     if (overrideColor) {
-        color = overrideColor;
+        color = overrideColor + ".main";
```

Comment:
I don't understand the logic behind this

Reply:
Shouldn't the color be .contrastText?

Reply:
When you choosing a bgColor (example secondary), it's color is secondary.constractText automatically.

Reply:
I make it for Kien's task. In his task about List, he must using bgColor is primary but color is secondary. So I make overrideColor for exceptions. If you don't using overrideColor, color is always onA when bgColor is A.

Reply:
![image](https://user-images.githubusercontent.com/58205909/118501834-50c21f80-b753-11eb-8c69-432b62333933.png)